backend/app/src/main/java/com/moh/phlat/backend/esb/boundary/PlrToken.java

```diff
@@ -0,0 +1,31 @@
+package com.moh.phlat.backend.esb.boundary;
+
+import java.time.Instant;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class PlrToken {
+	
+	private Instant lastRequestTime = Instant.now();
+	@Getter
```

Comment:
This would expose lastRequestTime though, right? Not that I think that's a huge issue, but regardless it would change the visibility of one of these properties.

Reply:
ok, leave it if it remains to be hidden from external access